naming/src/main/java/com/alibaba/nacos/naming/remote/rpc/handler/SubscribeServiceRequestHandler.java

```diff
@@ -93,6 +92,6 @@ public SubscribeServiceResponse handle(SubscribeServiceRequest request, RequestM
     @Deprecated
     private ServiceInfo handleClusterData(ServiceInfo data, ServiceMetadata metadata, Subscriber subscriber) {
```

Comment:
Whether this method can removed? 

I see the only used has been refactor in your changes.

Reply:
Yes, this logic has been contained in `selectInstancesWithHealthyProtection`.